connector-proxy-demo/connector-example/src/connector_example/commands/async_start.py

```
@@ -0,0 +1,109 @@
+                "mimetype": "application/json",
+            },
+            "command_response_version": 2,
+            "error": None,
+        }
+        body = json.dumps(callback_payload).encode("utf-8")
+        callback_request = request.Request(
+            callback_url,
+            data=body,
+            headers={"Content-Type": "application/json"},
+            method="PUT",
+        )
```

Ruff (0.15.18)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)